.github/workflows/main.yml

```diff
@@ -12,29 +12,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        build: [linux-debug, linux-release, windows-debug, windows-release]
+        build: [macos-debug, macos-release, linux-debug, linux-release, windows-debug, windows-release]
         include:
           - build: linux-debug
             os: ubuntu-latest
             config: debug
           - build: linux-release
             os: ubuntu-latest
             config: release
-          # - build: macos-debug
-          #   os: macos-latest
-          #   config: debug
-          # - build: macos-release
-          #   os: macos-latest
-          #   config: release
+          - build: macos-debug
+            os: self-hosted
```

Comment:
what does this mean?

Reply:
Did some googling, looks like this means you can self host your own github job runners, which I guess you were using for testing and don't actually want to merge in here?

Reply:
Yes it's what I use in my fork and is something that I don't want to merge